interpreter/apmint/hash_test.go

```diff
@@ -21,9 +21,24 @@ func TestLibpfEBPFFrameMarkerEquality(t *testing.T) {
 
 func newPythonTrace() *libpf.Trace {
 	trace := &libpf.Trace{}
-	trace.AppendFrame(libpf.NativeFrame, libpf.NewFileID(0, 0), 0)
-	trace.AppendFrame(libpf.NativeFrame, libpf.NewFileID(1, 1), 1)
-	trace.AppendFrame(libpf.NativeFrame, libpf.NewFileID(2, 2), 2)
+	trace.Frames.Append(&libpf.Frame{
+		Type:            libpf.NativeFrame,
```

Comment:
Not directly related to this change, but the function is called `newPythonTrace()` while it only returns frames of type `NativeFrame`.

Reply:
Fixed also in edac136fd598cccbad36a6c81e0f4e8d23335539.